charts/redpanda/templates/_configmap.go.tpl

```diff
@@ -0,0 +1,31 @@
+{{- /* Generated from "configmap.tpl.go" */ -}}
+
+{{- define "redpanda.RedpandaAdditionalStartFlags" -}}
```

Comment:
Just realized what bothers me about these files. Are we not supposed to indent?

Reply:
They're generated files, I didn't really intend for them to be human readable. Because everything is just JSON, there's no significance to the white space.

You're welcome to implement pretty printing, it might be a bit trickier than you think though.